src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -46,6 +46,132 @@
             <column name="last_modified_date" type="timestamp"/>
         </createTable>
 
+<!--        Address-->
+
+        <createTable tableName="address_state">
+            <column name="id" type="bigint" >
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="state" type="varchar(20)" />
+        </createTable>
+
+        <loadData tableName="address_state" file="config/liquibase/data/address_state.csv" separator=";">
+            <column name="id" type="numeric" />
+            <column name="state" type="string" />
+        </loadData>
+
+        <createTable tableName="address">
+            <column name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="street_address" type="varchar(50)" />
+            <column name="unit" type="varchar(20)" />
+            <column name="city" type="varchar(20)" />
+            <column name="state" type="bigint" />
```

Comment:
What do you think about calling this field state_id so we know it is the id from another table instead of the actual state itself?

Reply:
Good catch! This does need to be state_id